src/jit/codegenlinear.h

```diff
@@ -163,6 +163,7 @@ void genCodeForShiftLong(GenTreePtr tree);
 
 #ifdef _TARGET_XARCH_
 void genCodeForShiftRMW(GenTreeStoreInd* storeInd);
+void genCodeForBT(GenTreeOp* bt);
```

Comment:
You may want to call this BitTest rather than BT, since we will likely want this for other architectures eventually.

Reply:
ARM doesn't seem to have anything like BT. ARM64 has TBZ/TBNZ but that's quite different from BT and I'm not even sure how we can represent that in JIT's IR, it's a rather special case of conditional branch, maybe a JTRUE with a contained TEST_EQ/NE operand.

Reply:
Arm64 has tst:

      tst     w24, #24
      beq     G_M55607_IG12

https://github.com/dotnet/coreclr/pull/13799

Reply:
ARM64's `tst` is similar to x64's `test` and represented in IR by `GT_TEST_EQ` and `GT_TEST_NE`. `BT` is a different thing.

Reply:
What is the difference between these two? 
Any bit  (0-63) can be tested using Arm64 TST instruction. 

 The main difference seems to be that the Intel instruction sets the CF and the Arm instruction sets the ZF.  (And that a memory operand is allowed on Intel)

Reply:
> Any bit (0-63) can be tested using Arm64 TST instruction.

Yes, but the bit to test is specified by an immediate value. Presumably you can do something like `tst w24, w23` but then `w23` needs to contain a bit mask, not a bit index like in the case of `BT`.